File: i2f-jdk/i2f-swl/src/main/java/i2f/swl/impl/supplier/SwlSha256MessageDigesterSupplier.java
package i2f.swl.impl.supplier;

import i2f.swl.impl.SwlSha256MessageDigester;
import i2f.swl.std.ISwlMessageDigester;
import i2f.swl.std.supplier.ISwlMessageDigesterSupplier;

/**
 * @author Ice2Faith
 * @date 2026/4/8 15:13
 * @desc
 */
public class SwlSha256MessageDigesterSupplier implements ISwlMessageDigesterSupplier {
    @Override
    public ISwlMessageDigester get() {
        return new SwlSha256MessageDigester();
    }
}

File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-swl/core/impl/supplier/SwlSha256MessageDigesterSupplier.js
import SwlSha256MessageDigester from "../SwlSha256MessageDigester";

/**
 * @return {SwlSha256MessageDigesterSupplier}
 * @implements ISwlMessageDigesterSupplier
 */
function SwlSha256MessageDigesterSupplier() {

}

/**
 * @return {ISwlMessageDigester}
 */
SwlSha256MessageDigesterSupplier.prototype.get = function () {
    return new SwlSha256MessageDigester()
}

export default SwlSha256MessageDigesterSupplier
